packages/extension/src/background/index.ts

```diff
@@ -1,5 +1,17 @@
 import { browser, Runtime } from 'webextension-polyfill-ts';
 import { getBootData } from '@dailydotdev/shared/src/lib/boot';
+import { apiUrl } from '@dailydotdev/shared/src/lib/config';
+import {
+  ADD_BOOKMARKS_MUTATION,
+  CANCEL_UPVOTE_MUTATION,
+  REMOVE_BOOKMARK_MUTATION,
+  REPORT_POST_MUTATION,
+  UPVOTE_MUTATION,
+} from '@dailydotdev/shared/src/graphql/posts';
+import request from 'graphql-request';
+import { ADD_FILTERS_TO_FEED_MUTATION } from '@dailydotdev/shared/src/graphql/feedSettings';
+import { UPDATE_USER_SETTINGS_MUTATION } from '@dailydotdev/shared/src/graphql/settings';
+import { BOOT_LOCAL_KEY } from '@dailydotdev/shared/src/contexts/BootProvider';
 
 const cacheAmplitudeDeviceId = async ({
```

Comment:
This is no longer needed I think.
It should have been removed during the third party tracking task but we probably forgot

Reply:
Refactoring in separate PR